sunpy/map/sources/sdo.py

```diff
@@ -48,11 +48,15 @@ def __init__(self, data, header, **kwargs):
 
         # Fill in some missing info
         self.meta['detector'] = self.meta.get('detector', "AIA")
+        if 'bunit' not in self.meta and 'pixlunit' in self.meta:
+            self.meta['bunit'] = self.meta['pixlunit']  # PIXLUNIT is not a FITS standard keyword
         self._nickname = self.detector
         self.plot_settings['cmap'] = self._get_cmap_name()
         self.plot_settings['norm'] = ImageNormalize(
             stretch=source_stretch(self.meta, AsinhStretch(0.01)), clip=False)
-        # DN/s is not a FITS standard unit, so convert to counts/second
+        # DN is not a FITS standard unit, so convert to counts
```

Comment:
```suggestion
        # DN and DN/s are not FITS standard units, so convert to counts
```

Reply:
I'd argue that it's really just `DN` that's non-standard and that any composite unit containing DN is then obviously non-standard